src/main/resources/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep/help-credentials.html

```diff
@@ -0,0 +1,15 @@
+<div>
+    List of credentials to be used by the sshagent step.
+    The Pipeline Syntax Snippet Generator will guide the user to select a single credential from the list of available, like this:
+    <code>
+    sshagent(credentials: ['my-credential-id']) {
+        // some block
+    }
+    </code>
```

Comment:
I didn't check how the text appears before I proposed my change.  The `<code>` markup does not preserve the line breaks.  We either need explicit `<br/>` or we need the sample to be in a `<pre>` block.
```suggestion
<pre>
sshagent(credentials: ['my-credential-id']) {
    // some block
}
</pre>
```

With that change, now it looks like this:

![screencapture-localhost-8080-jenkins-pipeline-syntax-2021-04-24-12_06_28](https://user-images.githubusercontent.com/156685/115968532-a4936d00-a4f5-11eb-927c-80fad13d95af.png)